pillar.example

```diff
@@ -486,6 +486,11 @@ zoomdata:
 # In that case, consider to configure the options above.
 
 postgres:
+  # Make sure we're installing PostgreSQL 12
+  version: 12
```

Comment:
Does dependent `postgres-formula` already support PG 12? Nice.

Reply:
Yes - the corresponding changes were done to the bootstrap solution as well as binary data upgrade state.